LayoutTests/http/wpt/navigation-api/transition-promises-expected.txt

```diff
@@ -0,0 +1,5 @@
+CONSOLE MESSAGE: Unhandled Promise Rejection: Error
+
+PASS NavigationTransition finished promise is fulfilled on success
```

Comment:
Why don't we make an actual WPT test instead of making a WebKit-only test that uses WPT infrastructure?  Are there really 0 WPT tests that cover this?  We should fix that.

Reply:
There are many WPT tests that cover it, however they are very complex tests that require like 8 events happening and we currently fail in another area.

So this just extracts the one event so it has clear coverage until the others are complete.